android/src/main/java/io/sentry/capacitor/SentryCapacitor.java

```diff
@@ -129,4 +134,20 @@ public void startWithOptions(final PluginCall capOptions) {
 
         capOptions.resolve();
     }
+
+    @PluginMethod
+    public void captureEnvelope(PluginCall call) {
+        String envelope = call.getString("envelope");
+        try {
+            File installation =  new File(sentryOptions.getOutboxPath(), UUID.randomUUID().toString());
+            try (FileOutputStream out = new FileOutputStream(installation)) {
+                out.write(envelope.getBytes(Charset.forName("UTF-8")));
+            }
```

Comment:
1) Do we want to catch and log an error here if writing fails?
2) Is there any debug logger that we want here for when its successfully written?

Reply:
@joshdholtz  Updated!